ValkeyJSON.md

```diff
@@ -0,0 +1,914 @@
+    * Null if the document key does not exist.
+    * Null if the JSON path is invalid or does not exist.
+
+### ACL
+
+ValkeyJSON introduces a new ACL category - @json. The category includes all JSON commands. No existing Valkey commands 
+are members of the @json category. 
+
+There are 4 existing ACL categories which are updated to include new JSON commands: @read, @write, @fast, @slow. The 
+table below has a column for each of these categories and a row for each command. If the cell contains a “y” then that 
+command must be added into that category. All other command members of those categories remain unchanged.
+
+| JSON Command   | @json | @read | @write | @fast | @slow |
+|:---------------|:------|:------|:-------|:------|:------|
+| JSON.ARRAPPEND | y     |       | y      | y     |       |
+| JSON.ARRINDEX  | y     | y     |        | y     |       |
+| JSON.ARRINSERT | y     |       | y      | y     |       |
+| JSON.ARRLEN    | y     | y     |        | y     |       |
+| JSON.ARRPOP    | y     |       | y      | y     |       |
+| JSON.ARRTRIM   | y     |       | y      | y     |       |
+| JSON.CLEAR     | y     |       | y      | y     |       |
+| JSON.DEBUG     | y     | y     |        |       | y     |
+| JSON.DEL       | y     |       | y      | y     |       |
+| JSON.FORGET    | y     |       | y      | y     |       |
+| JSON.GET       | y     | y     |        | y     |       |
+| JSON.MGET      | y     | y     |        | y     |       |
+| JSON.MSET      | y     |       | y      |       | y     |
+| JSON.NUMINCRBY | y     |       | y      | y     |       |
+| JSON.NUMMULTBY | y     |       | y      | y     |       |
+| JSON.OBJKEYS   | y     | y     |        | y     |       |
+| JSON.OBJLEN    | y     | y     |        | y     |       |
+| JSON.RESP      | y     | y     |        | y     |       |
+| JSON.SET       | y     |       | y      |       | y     |
+| JSON.STRAPPEND | y     |       | y      | y     |       |
+| JSON.STRLEN    | y     | y     |        | y     |       |
+| JSON.TOGGLE    | y     |       | y      | y     |       |
+| JSON.TYPE      | y     | y     |        | y     |       |
+
+### Info Metrics
+
+Info metrics are visible through the “info json” or “info modules” command.
+
+| Info Name               | 	Description                                                    |
+|:------------------------|:------------------------------------------------------------------|
+| json_total_memory_bytes | Total amount of memory allocated to JSON documents and meta data. |
+| json_num_documents      | Number of JSON keys.                                              | 
+
+### Module Configs
+
+| Config Name            | Default Value | Unit | 	Description                                          |
+|:-----------------------|:--------------|:-----|:------------------------------------------------------|
+| json.max-document-size | 64            | MB   | Maximum memory allowed for a single JSON document.    |
```

Comment:
Please add an open question about what we want to do about this config. We want to be compatible with RedisJSON. Either:
1. Make the default value 0.
2. Remove the config until users ask for it.

Reply:
added the open question.

Reply:
Can you move it to its own section of its doc?